### PR TITLE
Created a file with the constituencies in Finland, and a README file …

### PR DESCRIPTION
**Pull request by marisviana**
…with more information and context around these changes

### CODE REVIEW

**Comment by marisviana** [comment]
@jloutsenhizer Hey Justin, could you please take a look at the proposed changes for Finland? Thanks!

**Comment by sguenther85** [comment]
Hi Mari,

ocd-division/country:fi/vaalipiirit:helsinki,Helsinki

we should use/prefer "ed" for electoral districts instead of "vaalipiirit". 
Source: https://en.wikipedia.org/wiki/Constituencies_of_Finland

The rest looks good!

**Comment by marisviana** [comment]
Thanks Steve, just updated the file.

**Comment by sguenther85** [comment]
Maybe rename the vaalipiirit.csv now to electoral_districts.csv like other ocd country files and then also sort the ocd paths alphabetically.

After that: @jloutsenhizer @jpmckinney do you two have anything else?
would be great if we could use the file for the election

**Comment by marisviana** [comment]
Sounds good, thanks! I just applied these changes and I think now we are good to go. Just waiting for the reviewers to take a look, and as Steve mentioned, the idea is to use these for the upcoming elections.

File: identifiers/country-fi/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This folder lists the following [constituencies of Finaland](https://en.wikipedia.org/wiki/Constituencies_of_Finland):
```
Comment (jloutsenhizer) [inline identifiers/country-fi/README.md:1]: Could you update the READEME to cover the other files as well?

File: identifiers/country-fi/electoral_districts.csv [hunk 2]
```diff
@@ -0,0 +1,14 @@
+id,name
+ocd-division/country:fi/ed:helsinki,Helsinki
+ocd-division/country:fi/ed:åland,Åland
+ocd-division/country:fi/ed:haeme,Häme
```
Comment (jloutsenhizer) [inline identifiers/country-fi/electoral_districts.csv:4]: Is there a reason to have the OCD ID use haeme instead of häme?

**Review by jloutsenhizer — CHANGES_REQUESTED** [review]
Overall looks good, just a couple comments

**Comment by marisviana** [comment]
Sounds good, thanks! I just updated the README file and applied your suggestions with regards to the OCD ID (makes more sense).

**Review by jloutsenhizer — APPROVED** [review]

**Review by jpmckinney — APPROVED** [review]
I approve.

Sidenote: I assume the numbers for the districts on Wikipedia are just to assist in reading the map, and are not an official number for each district (I don't see any evidence to that).

**Pull request merged**